pkg/api/norman/customization/logging/action.go

```diff
@@ -171,7 +181,7 @@ func (h *Handler) dryRunLoggingTarget(apiContext *types.APIContext, level, clust
 
 	podLister := context.Core.Pods(loggingconfig.LoggingNamespace).Controller().Lister()
```

Comment:
This PR is different from the 2.4 one. Doesn't this line need to change to `metav1.NamespaceAll` or is master different ? Same for change to user/controllers.go in other PR.